app/src/test/java/org/simple/clinic/deeplink/DeepLinkUpdateTest.kt

```diff
@@ -30,7 +30,8 @@ class DeepLinkUpdateTest {
   fun `if there is a logged in user and patient uuid is null, then show no patient error`() {
     val user = TestData.loggedInUser(
         uuid = UUID.fromString("dc0a9d11-aee4-4792-820f-c5cb66ae5e47"),
-        loggedInStatus = User.LoggedInStatus.LOGGED_IN
+        loggedInStatus = User.LoggedInStatus.LOGGED_IN,
+        capabilities = User.Capabilities(User.CapabilityStatus.Yes)
     )
```

Comment:
What do you think of creating two different users as class properties:

```kotlin
val medicalOfficer = // Create user who can record teleconsult
val nurse = // Create user who cannot record teleconsult
```

We can then use these personas in the individual tests.